genesyscloud/architect_flow/resource_genesyscloud_flow.go

```diff
@@ -113,10 +113,27 @@ func updateFlow(ctx context.Context, d *schema.ResourceData, meta interface{}) d
 	}
 
 	s3Uploader := files.NewS3Uploader(reader, nil, substitutions, headers, "PUT", presignedUrl)
-	_, err = s3Uploader.Upload()
+	fileInfo, err := os.Stat(filePath)
 	if err != nil {
+		log.Printf("failed to read file information. Path: '%s' Error: %v", filePath, err)
+	}
+
+	uploadErr := util.WithRetries(ctx, 20*time.Second, func() *retry.RetryError {
+		uploadStartTime := time.Now()
+		_, err = s3Uploader.Upload()
+		uploadDuration := time.Since(uploadStartTime)
```

Comment:
minor one : move this to inside error block

Reply:
Done